### DISCUSSION TITLE
How would i make a sonic mania styled save select screen

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#82] [Q&A]
I am trying to make a save select screen similar to sonic mania do you have any idea on how to make it

**Comment by DimensionWarped** [comment]
Hey. No more requests like this. It's getting on the devs nerves and we aren't here to teach people specifically how to implement things that are tied to presentation. Just focus on learning the engine. Make a level. You'll learn a *lot* by making a level, I promise. The next time you start a thread I want it to be because you have something you want to show off or if you are having trouble implementing something specific. I already mentioned before that you should have a detailed list of things you've already done before asking questions. If you want to talk to me personally you know how to reach me... just stop spamming the discussions boards.

**Comment by Renhoex** [comment]
I wanna give some advice here, I notice a lot of the changes you've been asking for (outside of the special stage request) are mostly aesthetic questions,
I'd say just, play around with what stuff is already here and see if you actually like working with the framework, this framework probably isn't the best for someone starting out as sonic physics can be really advanced to understand how it works.

My best advice here is that if you're struggling with this you should search these kind of questions up first like "godot how to save" or looking through the many other tutorials online, like see how other people get their mechanics going.

**Reply by Raptorq1** [reply]
Okay sorry
